src/main/groovy/com/readytalk/gradle/js/tasks/AsarPack.groovy

```diff
@@ -0,0 +1,23 @@
+package com.readytalk.gradle.js.tasks
+
+import com.moowork.gradle.node.task.NodeTask
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.OutputFile
+
+class AsarPack extends NodeTask {
+  @InputDirectory
+  def File inputDir
+
+  @OutputFile
+  def File outputFile
+
+  @Override
+  void exec() {
+    outputFile.parentFile.mkdirs();
+
+    setScript(project.file('node_modules/asar/bin/asar'))
```

Comment:
What are your thoughts on reading the package.json to check for the asar dependency? If there is none defined, either log a helpful error and exit, or log a warning and then install correct version of the dependency from Gradle.